testapp/testapp/testmain/models.py

```diff
@@ -0,0 +1,22 @@
+# Create your models here.
+from decimal import Decimal
+
+from payments import PurchasedItem
+from payments.models import BasePayment
+
+
+class Payment(BasePayment):
+    def get_failure_url(self):
+        return "http://example.com/failure/"
+
+    def get_success_url(self):
+        return "http://example.com/success/"
```

Comment:
Ditto, but for successful payments.

Reply:
Should I create generic views or leave it customizable from settings? :thinking: 

Maybe a generic view by default would be a simpler approach.

Reply:
A generic view si fine. This is just so we can use this app to manually test providers, so it's just so we can do 100% of the flow via a gui.